internal/constants/errors.go

```diff
@@ -0,0 +1,7 @@
+package constants
```

Comment:
We could have a `constants` and `errors` folder? But happy with this regardless

Reply:
Yeah I wasn't sure about this either. I kinda just wanted a central place where all of these more "global" constants could be defined. I think for now I'll keep it as is but try to think of a better naming standard for it.